doc/JTS_Version_History.html

```diff
@@ -15,22 +15,77 @@ <h1 style='text-align:center; color:darkblue;'>
 This document lists the change history of release versions of the
 <span style='color:darkblue;'><b>JTS Topology Suite</b></span>.
 <p>
-Further information on JTS can be found at <b>Tsusiat Software's
-<a href='http://tsusiatsoftware.net/jts/main.html'>JTS pages</a></b>.
+The current JTS distributions can be obtained at the
+<a href='https://github.com/locationtech/jts/releases'><b>Github JTS site</b></a>.
+
 <p>
-The current and archived JTS distributions can be obtained at the
+The JTS distributions for older versions can be obtained at the
 <a href='http://sourceforge.net/projects/jts-topo-suite/'><b>SourceForge JTS site</b></a>.
 
 <!-- ================================================================ -->
 <hr size='4' color='darkblue'>
 <h2>Version 1.15</h2>
 
-<i>Release Date:  September 23, 2015</i>
+<i>Release Date:  November 30, 2017</i>
+
+<h3>Project Changes</h3>
+<ul>
+<li>Changed licensing to dual-licensed EDL or EPL
+<li>Changed source hosting to GitHub
+<li>Changed distro hosting to GitHub
```

Comment:
We also have hosting downloads at locationtech, is this doable @jnh5y ?

Reply:
@jodygarnett 'yes', it is doable.  I haven't uploaded zips to LocationTech since GeoMesa 1.2.0, so I'm not sure how hard it is and if anything has changed.